### PR TITLE
Build/improve ontology documentation

### PR DESCRIPTION
**Pull request by ccamel**
Partially resolve issue https://github.com/okp4/docs/issues/407 by delivering the following elements:

- A separate documentation file for each Verifiable Credential schema.
- A list of all namespaces utilized by the schema.
- Detailed documentation of every class defined within the schema.
- Detailed documentation of all properties associated with the class as defined in the schema.
- Documentation of examples, where available.

Preview:
- https://github.com/okp4/ontology/blob/9e3341854b3c7c0fb50ade4307c5fea8d4a06bdf/docs/schemas/credential-dataset-description.md
- https://github.com/okp4/ontology/blob/9e3341854b3c7c0fb50ade4307c5fea8d4a06bdf/docs/schemas/credential-governance-text.md

## Summary by CodeRabbit

- **New Features**
	- Introduced several new schema documentation for credentials, enhancing the understanding and application of datasets, digital resources, services, governance texts, and zones.
	- Added a new `output_path` argument in CLI for specifying documentation generation paths.
	- Implemented a structured markdown document generation from schema graphs in CLI templates.

- **Documentation**
	- Updated and expanded documentation to include comprehensive details on new and existing schemas.

- **Bug Fixes**
	- Corrected permissions setting command for Markdown files to ensure proper documentation generation.

- **Refactor**
	- Updated variable and function signatures in Makefile and CLI scripts for improved documentation generation and linting processes.
	- Renamed classes and updated references in the digital resource schema for consistency.

- **Chores**
	- Enhanced CLI documentation generation capabilities with new parsing and rendering functions.

### CODE REVIEW

File: src/schema/digital-resource/credential-digital-resource-rights.ttl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 @prefix : <https://w3id.org/okp4/ontology/v$major/schema/credential/digital-resource/rights/> .
 @prefix schema: <http://schema.org/> .
 
-:DigitalResourceRightsCredentials a rdfs:Class ;
+:DigitalResourceRightsCredential a rdfs:Class ;
```
[the remark this answers is not in the document]
Comment (ccamel) [reply src/schema/digital-resource/credential-digital-resource-rights.ttl:6]: Oh, good one. I will correct this. :relieved:

**Comment by ccamel** [comment]
@coderabbitai pause

**Review by MdechampG — APPROVED** [review]
Much more user-friendly thanks ! 😊

**Review by amimart — APPROVED** [review]
That's better thx 😄

**Pull request merged**